corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -671,20 +691,26 @@ ComboboxEntry.prototype.onPreProcess = function (newValue) {
         this.question.error(gettext('Not a valid choice'));
     }
 };
-ComboboxEntry.prototype.receiveValue = function (value) {
+ComboboxEntry.prototype.receiveMessage = function (message, field) {
+    // Iterates through options and selects an option that matches message[field].
+    // Registers a no answer if message[field] is not in options.
+    // Also accepts fields in format `field1||field2||...||fieldn` it will find the
```

Comment:
What's the scenario where a `||`-delimited string is sent to this function?

Reply:
As in why would the user want `||` in the string? That's to provide fallback option when a field is not present (e.g. `us_state` when it is not a state) or when the first `message[field]` does not match any of the options.